_posts/2018/02/2018-02-28-carpentrycon-accomm.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+authors: ["Belinda Weaver", "Fotis Psomopoulos"]
+title: "CarpentryCon - Hotel Accommodation Options"
+date: 2018-02-28
+time: "0:00:00"
+category: [ "CarpentryCon", "Accommodation", "Conferences"]
+---
+
+## Hotel Accommodation for CarpentryCon 2018
+
+We have two hotel options for you for [CarpentryCon 2018](http://www.carpentrycon.org/). These hotels are offering us a special CarpentryCon rate.
+
+### [Clayton Hotel, Ballsbridge](https://www.claytonhotelballsbridge.com)
+
+- [See it on the Dublin map](https://www.google.com.au/maps/place/Clayton+Hotel+Ballsbridge/@53.3256826,-6.2271519,17z/data=!3m1!4b1!4m5!3m4!1s0x48670ec57aa7b09d:0x7965235d57af13f!8m2!3d53.3256826!4d-6.2249632?hl=en).
+- [Book online](https://www.claytonhotelballsbridge.com) or by calling the hotel + 353 1 668 1111.
+- **Reservation Code:** CARP300518.
+
```

Comment:
Both hotels have a cut-off date of April 18th. We should make this also clear on the "introduction", i.e. "The special CarpentryCon rates will be valid for the period of the event and until the cut-off date; anyone looking to book after this will be offered the best available rate at the time."